### PR TITLE
Clarify `to_openmm_simulation` docstring and rename `to_lammps(file_path)` arg to `prefix`

### PR DESCRIPTION
**Pull request by Yoshanuikabundi**
#1027

### CODE REVIEW

File: openff/interchange/components/interchange.py [hunk 1]
```diff
@@ -413,27 +413,21 @@ def to_gro(self, file_path: Path | str, decimal: int = 3):
...
-        # TODO: Rename `file_path` to `prefix` (breaking change)
-        prefix = str(file_path)
-        if prefix.endswith(".lmp"):
-            prefix = prefix[:-4]
-
-        datafile_path = prefix + ".lmp"
+        datafile_path = str(prefix) + ".lmp"
         self.to_lammps_datafile(datafile_path, writer=writer)
         self.to_lammps_input(
             prefix + "_pointenergy.in",
```
Comment (mattwthompson) [inline openff/interchange/components/interchange.py]:
```suggestion
            str(prefix) + "_pointenergy.in",
```

**Comment by Yoshanuikabundi** [comment]
@mattwthompson I think this is ready!

File: examples/lammps/lammps.ipynb [hunk 2]
```diff
@@ -87,14 +86,14 @@
...
-    "interchange.to_lammps(\"interchange.lmp\")"
+    "interchange.to_lammps_datafile(\"interchange.lmp\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we need to write an input file for LAMMPS. Parts of these input files depend on force field parameters, so we should use a sample input file written for our interchange as a starting point. We can generate such a sample file from `MDConfig`:"
+    "Now we need to write an input file for LAMMPS. Parts of these input files depend on force field parameters, so we should use a sample input file written for our interchange as a starting point. We can generate such a sample file with the `to_lammps_input()` method:"
```
Comment (mattwthompson) [inline examples/lammps/lammps.ipynb:96]: 🎉

File: openff/interchange/components/interchange.py [hunk 3]
```diff
@@ -574,7 +578,7 @@ def to_openmm_simulation(
             If True, add valence forces that might be overridden by constraints, i.e. call `addBond` or `addAngle`
             on a bond or angle that is fully constrained.
         additional_forces : Iterable[openmm.Force], default=tuple()
-            Additional forces to be added to the system, i.e. barostats that are not
+            Additional forces to be added to the system, eg. barostats, that are not
```
Comment (mattwthompson) [inline openff/interchange/components/interchange.py:581]:
```suggestion
            Additional forces to be added to the system, e.g. barostats, that are not
```

**Review by mattwthompson — APPROVED** [review]
LGTM, thanks for fixing the tests!

**Pull request merged**